specs/app.spec.js

```diff
@@ -0,0 +1,149 @@
+const fs = require('fs'),
+    mysql = require('mysql'),
+    parse = require('csv-parse'),
+    expect = require('chai').expect,
+    DB_CONFIG = require('../dbconfig.json'),
+    sinon = require('sinon');
+
+describe('When the user has csv files in the files folder', function () {
+    this.timeout(10000);
+
+    beforeEach(function () {
+        return ensureTablesRemoved(['t1', 't2', 'tr']);
```

Comment:
Why are you ensuring that t1, t2 and tr are removed? I can only see t1 inside the files folder. All assets required to run tests should be included in source control.